examples/directml/llm/run_phi2.py

```diff
@@ -0,0 +1,28 @@
+
```

Comment:
This can be removed. We should be able to use the same inference script as the other models.